### PR TITLE
add a plist to the chromedriver formula that allows launchd autostart

### PR DESCRIPTION
**Pull request by dreed1**
we've been working on getting our dev and CI environments to have greater parity... and since our CI server automatically starts a chromedriver we thought it'd be a good idea to have our dev environments do the same.
We're using launchd with a few other brew formulae, and we wanna keep with that pattern so we went ahead and added a plist to this formula. thanks!

### CODE REVIEW

File: Library/Formula/chromedriver.rb [hunk 1]
```diff
@@ -9,4 +9,30 @@ class Chromedriver < Formula
...
+    <dict>
+      <key>Label</key>
+      <string>homebrew.mxcl.chromedriver</string>
+      <key>RunAtLoad</key>
+      <true/>
+      <key>KeepAlive</key>
+      <false/>
+      <key>ProgramArguments</key>
+      <array>
+        <string>/usr/local/bin/chromedriver</string>
```
Comment (mikemcquaid) [inline Library/Formula/chromedriver.rb]: This should use `opt_bin` rather than hardcoding `/usr/local`.

**Comment by mikemcquaid** [comment]
One nit but otherwise looking good!

**Comment by tdsmith** [comment]
@BrewTestBot test this please

**Comment by dreed1** [comment]
i feel weird about having to merge so many times, let me know if you want me to squash or resubmit this in a new pull request.

**Comment by mikemcquaid** [comment]
Yeh, can you squash and rebase and then shout and I'll pull. Sorry for the delay!

**Comment by dreed1** [comment]
:+1: sounds good, i'll get to that in a bit here. thx!

**Comment by mikemcquaid** [comment]
Closing in favour of #37122.

**Pull request closed without merging**